api/src/test/java/org/openmrs/PatientTest.java

```diff
@@ -10,18 +10,18 @@
 package org.openmrs;
 
 import static org.hamcrest.Matchers.contains;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.assertSame;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
its a minor thing, but we should group imports by packages. this can be configured in the IDE. `org.junit.jupiter` imports should be grouped together. `hamcrest` is mixed in there.